docs/registry.md

```diff
@@ -0,0 +1,60 @@
+ # Registry Support for SOCI
+
+SOCI is supported by most registries. The notable exception to this is Duckerhub, which currently does not meet our technical requirements. 
```

Comment:
"meet our technical requirements" sounds like we have some non-standard requirement. Is it possible to change to something like "which doesn't fully support OCI 1.0 used by SOCI".